modules/lib/build.gradle

```diff
@@ -8,11 +8,18 @@ node {
     version = '16.15.0'
 }
 
+def coreDir = "$projectDir/core/"
```

Comment:
It would be nice to NOT introduce an extra directory.
Can we have a core.ts file instead and not interfare with the build.
Or make core a gradle module

Reply:
We can, but we have a unique README file for this module. Where should it be placed? Just pile all in the `/lib`?

Reply:
As discussed, will be updated later in different task.